app/models/guide.rb

```diff
@@ -52,7 +52,8 @@ def pending_exercises(user)
                 on assignments.exercise_id = exercises.id
                 and assignments.submitter_id = #{user.id}
                 and assignments.submission_status = #{Mumuki::Domain::Status::Submission::Passed.to_i}").
-        where('assignments.id is null')
+        where('assignments.id is null').
+        order('public.exercises.number asc')
```

Comment:
This ordering logic it's related to `next_exercise`, not to `pending_exercises`.
It's an important distinction because we'll use indicators to easily get pending_items but ordering logic is not included there.

Reply:
Great, I had that doubt :smile:

Reply:
Please review it again!